src/charts.js

```diff
@@ -263,8 +264,27 @@ function requestData(url) {
     });
 }
 
+function getSingleUrl(pkg, fromDate, toDate) {
+  return 'https://api.npmjs.org' + '/downloads/range/' + dateToDayKey(fromDate) + ':' + dateToDayKey(toDate) + '/' + encodeURIComponent(pkg);
```

Comment:
I think, previously, this was a relative URL (see [line 267](https://github.com/pvorb/npm-stat.com/pull/47/files#diff-3a788d2133ba7ece804c0d5d9c9b4d6cL267)). You should be able to leave it as-is when developing using `npm run start`. Requests will be proxied.